routes/model_routes.py

```
@@ -17,7 +17,7 @@
     current_app,
 )
 from flask_login import login_required, current_user
-from database import db_connection  # Use the centralized context manager
+from database import get_db  # Use the centralized context manager
```

Ruff (0.8.2)

`database.get_db` imported but unused

Remove unused import: `database.get_db`

(F401)